cmd/ttn-lw-cli/commands/gateways.go

```diff
@@ -202,9 +205,27 @@ var (
 					return err
 				}
 			}
+
+			setDefaults, _ := cmd.Flags().GetBool("defaults")
```

Comment:
This is fine for now, but for the future: if you only need `setDefaults` once, we prefer writing it as

```go
if setDefaults, _ := cmd.Flags().GetBool("defaults"); setDefaults { 
    ...
```

Reply:
For remembering the correct practices, I corrected it as well.